tests/test_all.py

```diff
@@ -105,9 +105,10 @@ def test_product(auto_container):
     OS_VERSION != "tumbleweed",
     reason="product flavors only available for openSUSE",
 )
-def test_opensuse_product_flavor(auto_container):
+@pytest.mark.parametrize("container", CONTAINERS_WITH_ZYPPER, indirect=True)
```

Comment:
Just out of curiosity, why is this change necessary?

Reply:
I tried to explain it in there commit message.

> They wouldn't be able to report the flavor anyway

It is necessary because the micro (?) container doesn't have rpm(1) die the test fails for unrelated reasons. There's no variable that lists containers with rpm, and we don't need it as the flavor only matters for containers that can and are running zypper.